ui/src/tool-panel/location-track/location-track-infobox-duplicate-of.tsx

```diff
@@ -75,63 +142,99 @@ export const LocationTrackInfoboxDuplicateOf: React.FC<LocationTrackInfoboxDupli
 }: LocationTrackInfoboxDuplicateOfProps) => {
     const { t } = useTranslation();
     const trackNumbers = useTrackNumbers(layoutContext);
+    const explicitDuplicateLocationTrackNames = useLocationTracks(
+        duplicatesOfLocationTrack
+            ?.map((d) => d.duplicateStatus.duplicateOfId)
+            ?.filter(filterNotEmpty) ?? [],
+        layoutContext,
+    );
+    const duplicateTrackNumberWarning =
+        existingDuplicate && currentTrackNumberId !== existingDuplicate?.trackNumberId
+            ? t('tool-panel.location-track.duplicate-on-different-track-number', {
+                  trackNumber: getTrackNumberName(trackNumbers, existingDuplicate.trackNumberId),
+              })
+            : '';
+
     return existingDuplicate ? (
-        <React.Fragment>
+        <span title={duplicateTrackNumberWarning}>
             <LocationTrackLink
                 locationTrackId={existingDuplicate.id}
                 locationTrackName={existingDuplicate.name}
             />
             &nbsp;
             {currentTrackNumberId !== existingDuplicate.trackNumberId && (
-                <LocationTrackDuplicateTrackNumberWarning
-                    differingTrackNumberName={getTrackNumberName(
-                        trackNumbers,
-                        existingDuplicate.trackNumberId,
-                    )}
-                />
+                <LocationTrackDuplicateInfoIcon type={'ERROR'} />
```

Comment:
Tämä on parenttiduplikaatin näyttämisosuus. Näitä mahdollisia virheitä on tasan 1, joten laskin `LocationTrackDuplicateNotice`:n olevan vähän turhan hifistelyä. Tooltip kuitenkin refaktoroitu täälläkin siten, että se näkyy nimeäkin hoveroimalla